tools/bazel/postinstall-patches.js

```diff
@@ -119,8 +119,7 @@ searchAndReplace(`[formatProperty + "_ivy_ngcc"]`, '[formatProperty]',
 // Workaround for https://github.com/angular/angular/issues/33452:
 searchAndReplace(/angular_compiler_options = {/, `$&
         "strictTemplates": True,
-        "strictDomLocalRefTypes ": False,
-        "strictDomEventTypes": False,`, 'node_modules/@angular/bazel/src/ng_module.bzl');
+        "fullTemplateTypeCheck": True,`, 'node_modules/@angular/bazel/src/ng_module.bzl');
```

Comment:
It looks like `fullTemplateTypeCheck` is already enabled in the `ng_module` bazel rule ([for more context](https://github.com/angular/angular/blob/660546d0ec63d74cc8bee46498a117a36640bb48/packages/bazel/src/ng_module.bzl#L661)). If it would have been set to `false`, then the `strictTemplates` flag would be a noop too.